app/src/main/java/com/example/music_app/data/repositories/playlists/PlaylistsRepositoryImpl.kt

```diff
@@ -32,4 +33,25 @@ class PlaylistsRepositoryImpl(
             }
         )
     }
-}
\ No newline at end of file
+
+    override suspend fun createPlaylist(
+        name: String,
+        description: String?
+    ): Flow<Result<Boolean, AppErrors>> = flow {
+        val userIdResponse = spotifyAPI.getCurrentUser().id
```

Comment:
Could the userId be retrieved along side with the login and stored through the user session, so you won't have the necessity to get user id each time you make such requests ?